src/test/integration/completion.test.ts

```diff
@@ -4,64 +4,145 @@
+        new vscode.CompletionItem('variable', vscode.CompletionItemKind.Class),
+      ];
+      expected.push(...snippets);
+      await testCompletion(docUri, new vscode.Position(0, 0), {
+        items: expected,
+      });
+    });
+  });
+
+  suite('module completion', function suite() {
+    const docUri = getDocUri('main.tf');
+
+    this.beforeAll(async () => {
+      await open(docUri);
+      await activateExtension();
+    });
+
+    teardown(async () => {
+      await vscode.commands.executeCommand('workbench.action.closeAllEditors');
+    });
+
+    test('language is registered', async () => {
+      const doc = await vscode.workspace.openTextDocument(docUri);
+      assert.equal(doc.languageId, 'terraform', 'document language should be `terraform`');
+    });
+
+    // Completion for inputs of a local module
+    test('inputs of a local module', async () => {
+      const expected = [
+        new vscode.CompletionItem('count', vscode.CompletionItemKind.Property),
+        new vscode.CompletionItem('depends_on', vscode.CompletionItemKind.Property),
+        new vscode.CompletionItem('for_each', vscode.CompletionItemKind.Property),
+        new vscode.CompletionItem('machine_type', vscode.CompletionItemKind.Property),
+        new vscode.CompletionItem('providers', vscode.CompletionItemKind.Property),
+        new vscode.CompletionItem('version', vscode.CompletionItemKind.Property),
+      ];
+      expected.push(...snippets);
+
+      await testCompletion(docUri, new vscode.Position(21, 0), {
+        items: expected,
+      });
+    });
+
+    // Completion for a local module sources (prefix ./)
+    test('local module sources', async () => {
+      const expected = [
+        new vscode.CompletionItem('"./ai"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"./compute"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"cloudposse/label/null"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/eks/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/iam/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/kms/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/lambda/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/rds/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/s3-bucket/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/security-group/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-aws-modules/vpc/aws"', vscode.CompletionItemKind.Text),
+        new vscode.CompletionItem('"terraform-google-modules/project-factory/google"', vscode.CompletionItemKind.Text),
```

Comment:
I like that you've included the results from the module registry here, but I'm wondering if this might introduce some flakiness to this test. I think the results and order are based on the popularity of the modules in the registry. If another module becomes more popular, we will probably have to update this test.

To complete only local modules, we could move the completion position right after the `./` prefix (18, 14). This should give us two results.